src/PersonObjects/Sleeve/ui/StatsElement.tsx

```diff
@@ -1,31 +1,110 @@
-import { Sleeve } from "../Sleeve";
-import { numeralWrapper } from "../../../ui/numeralFormat";
 import React from "react";
 
-import { StatsTable } from "../../../ui/React/StatsTable";
+import {
+  Typography,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+} from "@mui/material";
+
+import { numeralWrapper } from "../../../ui/numeralFormat";
+import { Settings } from "../../../Settings/Settings";
+import { StatsRow } from "../../../ui/React/StatsRow";
+import { characterOverviewStyles as useStyles } from "../../../ui/React/CharacterOverview";
+import { Money } from "../../../ui/React/Money";
+import { MoneyRate } from "../../../ui/React/MoneyRate";
+import { ReputationRate } from "../../../ui/React/ReputationRate";
+import { use } from "../../../ui/Context";
+
+import { Sleeve } from "../Sleeve";
+import { SleeveTaskType } from "../SleeveTaskTypesEnum";
 
 interface IProps {
   sleeve: Sleeve;
 }
 
 export function StatsElement(props: IProps): React.ReactElement {
-  const rows = [
-    [
-      "HP: ",
-      <>
-        {numeralWrapper.formatHp(props.sleeve.hp)} / {numeralWrapper.formatHp(props.sleeve.max_hp)}
-      </>,
-    ],
-    ["City: ", <>{props.sleeve.city}</>],
-    ["Hacking: ", <>{numeralWrapper.formatSkill(props.sleeve.hacking)}</>],
-    ["Strength: ", <>{numeralWrapper.formatSkill(props.sleeve.strength)}</>],
-    ["Defense: ", <>{numeralWrapper.formatSkill(props.sleeve.defense)}</>],
-    ["Dexterity: ", <>{numeralWrapper.formatSkill(props.sleeve.dexterity)}</>],
-    ["Agility: ", <>{numeralWrapper.formatSkill(props.sleeve.agility)}</>],
-    ["Charisma: ", <>{numeralWrapper.formatSkill(props.sleeve.charisma)}</>],
-    ["Shock: ", <>{numeralWrapper.formatSleeveShock(100 - props.sleeve.shock)}</>],
-    ["Sync: ", <>{numeralWrapper.formatSleeveSynchro(props.sleeve.sync)}</>],
-    ["Memory: ", <>{numeralWrapper.formatSleeveMemory(props.sleeve.memory)}</>],
-  ];
-  return <StatsTable rows={rows} />;
+  const classes = useStyles();
+
+  return (
+    <Table sx={{ display: 'table', mb: 1, width: '100%' }}>
+      <TableBody>
+        <StatsRow name="City" color={Settings.theme.primary} data={{ content: props.sleeve.city }} />
+        <StatsRow name="HP" color={Settings.theme.hp}
+          data={{ content: `${numeralWrapper.formatHp(props.sleeve.hp)} / ${numeralWrapper.formatHp(props.sleeve.max_hp)}` }}
+        />
+        <StatsRow name="Hacking" color={Settings.theme.hack} data={{ level: props.sleeve.hacking, exp: props.sleeve.hacking_exp }} />
```

Comment:
this could be abstracted out think i saw these higher up in the pr already

Reply:
how so? the `<StatsRow>` component is already an abstraction itself heh